src/SprykerEco/Yves/Easycredit/Theme/default/views/easycredit/easycredit.twig

```diff
@@ -0,0 +1,58 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'form',
+    tag: 'div'
+} %}
+
+{% define data = {
+    form: required,
```

Comment:
You just copy-pasted whole data from form molecule? You are using only `form` and `legalText` in your molecule, please remove everything else.

Reply:
this view was created by Vladymyr Hrychenko, i will clean up it